#163713875 Fix database setup configuration

czarjulius opened the pull request:
#### What does this PR do?
Fixes the database migration error.

#### Description of Task to be completed?
Tables can be created and populated on the Postgres database.

#### How should this be manually tested?
when you run npm run mig, it will go ahead and create the tables.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#163713875
#### Screenshots or gifs (if appropriate)
#### Questions:

czarjulius committed:
bug(fixedMigrations) : installed @babel-node -ensure all fields captured [Delivers #163713875]

ascii-dev commented:
Hi @czarjulius, nice work you have here, but your commit does not follow the convention as it has no `body`. Also, why do your tests and coverage have issues? Thanks

ascii-dev commented on server/middlewares/jwtAuth.js:
```diff
@@ -0,0 +1,55 @@
+  },
+};
+
+export default auth;
```
You could've made this a class and the functions could be static methods in them. Kindly look into this. Thanks

ascii-dev commented on server/migration/deployment.js:
```diff
@@ -1,25 +1,35 @@
+//  isAdmin boolean DEFAULT VALUE false,
+//  regDate date NOT NULL DEFAULT CURRENT_DATE
+// );
+// `;
```
Any reason why you have a comment code block here?

ascii-dev commented on server/routes/index.js:
```diff
@@ -2,7 +2,8 @@ import express from 'express';
 import OfficeController from '../controllers/OfficeController';
-import AuthController from '../controllers/AuthUserController';
+//import AuthController from '../controllers/AuthUserController';
+//import validator from '../middlewares/validate';
```
Any reason why you have comment code blocks here?

czarjulius committed:
bug(feedback): Added functions into a single class
- ensure that all comments were cleaned up
- follow the commit message best practice [Delivers #163713875]

czarjulius committed:
bug(feedback): Added functions into a single class
- ensure that all comments were cleaned up
- follow the commit message best practice [Delivers #163713875]
